.github/workflows/pull-request-teardown.yml

```diff
@@ -24,15 +24,21 @@ jobs:
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
       with:
         kubectl: 1.16.2
-        helm: 2.16.1
+        helm: 3.3.1
         command: |
           export PR_NUMBER=${{ github.event.number }}
           echo $PR_NUMBER
           export RENKU_RELEASE="ci-${PR_NUMBER}-renku"
-          helm init --client-only
+          export RENKU_NAMESPACE="ci-${PR_NUMBER}-renku"
           echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
-          helm delete --purge $RENKU_RELEASE
-          helm delete --purge $RENKU_RELEASE-tmp-notebooks || true
+          helm uninstall $RENKU_RELEASE --namespace $RENKU_NAMESPACE
+          kubectl delete all --all --namespace $RENKU_NAMESPACE
+          kubectl delete --all secrets --namespace $RENKU_NAMESPACE
+          kubectl delete ns $RENKU_NAMESPACE
+          helm uninstall $RENKU_RELEASE-tmp-notebooks --namespace $RENKU_NAMESPACE-tmp || true
```

Comment:
I think the `|| true` is a legacy from the time when not all ci-deployments had anonymous notebooks enabled. I'd remove it to let the pipeline fail rather than report success while something has gone wrong during the removal of the tmp deployment/namespace.

Reply:
yes, I thought that we might still want to disable anonymous notebooks for some CIs, but if that's not the case I can just clean up those leftovers.